tests/test_fs.py

```diff
@@ -23,16 +23,77 @@
 from v3iofs.path import split_container
 
 
-def test_ls(fs: V3ioFS, new_file):
-    new_file(fs._client, f'{test_dir}/test-file')  # Make sure dir exists
+def test_ls(fs: V3ioFS, tmp_obj):
+
     path = f'/{test_container}/{test_dir}/'
-    out = fs.ls(path)
-    assert len(out) > 0, 'nothing found'
-    assert all(isinstance(p, dict) for p in out), 'not dict'
+    out0 = fs.ls(path)
+    assert len(out0) > 0, 'nothing found'
+
+    out1 = fs.ls(path, detail=False)
+    assert len(out1) > 0, 'nothing found'
+    assert all(isinstance(p, str) for p in out1), 'not string'
+    out1 = [f for f in out1 if not 'iguazio' in f]
+    assert set(out1) == set(
+        ['/bigdata/v3io-fs-test/test-file',
```

Comment:
Constuct the paths from `test_dir`, not hard coded. Even better approach will be to have `tmp_obj` contain the list of files & directrries (probably as a `dict`).

This applies to all cases below, including `test_glob`